napari/layers/labels/labels.py

```diff
@@ -534,13 +491,17 @@ def _update_thumbnail(self):
         """Update thumbnail with current image data and colors.
         """
         zoom_factor = np.divide(
-            self._thumbnail_shape[:2], self._image_view.shape[:2]
+            self._thumbnail_shape[:2], self._data_view.shape[:2]
         ).min()
-        downsampled = np.round(
-            ndi.zoom(self._image_view, zoom_factor, prefilter=False, order=0)
-        )
-        downsampled = self.raw_to_displayed(downsampled)
-        colormapped = self.colormap.map(downsampled)
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
Maybe add a comment that this warning filter can be removed with scipy 1.4?

Reply:
added